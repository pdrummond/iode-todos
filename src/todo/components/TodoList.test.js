import React from "react";
import renderer from "react-test-renderer";
import TodoList from "./TodoList";

describe("components", () => {
	describe("TodoList", () => {
		it("should render correctly", () => {
			const props = {
				todos: [
					{
						id: 1,
						text: "Todo 1",
						completed: false
					}
				],
				onTodoClick: jest.fn()
			};
			const tree = renderer.create(<TodoList {...props} />).toJSON();
			expect(tree).toMatchSnapshot();
		});
	});
});
