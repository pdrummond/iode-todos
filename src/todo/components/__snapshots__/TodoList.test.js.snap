// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components TodoList should render correctly 1`] = `
<ul>
  <div
    className="Todo"
    onClick={[Function]}
  >
    Todo 1
  </div>
</ul>
`;
